formatter_initializer.c

```diff
@@ -0,0 +1,28 @@
+#include <stddef.h>
+#include "main.h"
+/**
+ * initialise_formatters - Initializes the array of format specifiers
+ * @specification: Array of Formatter structures
+ */
+void initialise_formatters(Formatter *specification)
+{
+	int i = 0;
+
+	for (i = 0; i < 256; i++)
+	{
+		specification[i].symbol = '\0';
```

Comment:
@tingru0515 : 

You can try changing every initialisation of the form

```
specification[A].symbol = A;
specification[A].print = F;
```
to 

```
specification[A] = { symbol.A, print.B };
```
Or whatever the correct syntax is for the latter.
I tried doing `{ A, B }` but the compiler didn't like the compound declaration. Not sure if `{ member.value, member.value }` counts as compound.

Reply:
Ask @zonafrank about this. He may have tried it already.